src/app/modules/webhook/webhook.utils.ts

```diff
@@ -14,3 +20,30 @@ export const formatWebhookResponse = (
   headers: stringifySafe(response?.headers) ?? '',
   data: stringifySafe(response?.data) ?? '',
 })
+
+/**
+ * Computes epoch of next webhook attempt based on previous attempts.
+ * @param previousAttempts Array of epochs of previous attempts
+ * @returns ok(epoch of next attempt) if there are valid retries remaining
+ * @returns err(WebhookNoMoreRetriesError) if there are no more retries remaining
+ */
+export const getNextAttempt = (
+  previousAttempts: number[],
+): Result<number, WebhookNoMoreRetriesError> => {
+  if (previousAttempts.length >= RETRY_INTERVALS.length)
```

Comment:
nit: if return is already on the next line, should we add braces for better visual blocks